lib/Eclair/LLVM/HashMap.hs

```diff
@@ -35,7 +35,7 @@ data HashMap
 
 data CGState
   = CGState
-  { externals :: Externals
+  { _externals :: Externals
```

Comment:
I checked in the code, this field can be removed entirely. It only needs to passed into the `Vector.codegen` function.